Kickstarter-iOS/Views/Controllers/PledgeViewController.swift

```diff
@@ -412,6 +426,24 @@ final class PledgeViewController: UIViewController, MessageBannerViewControllerP
       self?.viewModel.inputs.scaFlowCompleted(with: status, error: error)
     }
   }
+
+  private func showProcessingView() {
+    guard let window = UIApplication.shared.keyWindow else {
+      return
+    }
+
+    _ = (self.processingView, window)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToEdgesInParent()
+
+    self.processingView.activityIndicator.startAnimating()
```

Comment:
I think we could start animating the indicator in the ProcessingView's init method. This way we don't need to expose its UIActivityIndicatorView property and it's one less thing to worry about, since we don't need to start/stop the animation.

Reply:
Good idea!